src/stdlib_strings.f90

```diff
@@ -4,13 +4,15 @@
 !>
 !> The specification of this module is available [here](../page/specs/stdlib_strings.html).
 module stdlib_strings
-    use stdlib_ascii, only : whitespace
-    use stdlib_string_type, only : string_type, char, verify
+    use stdlib_ascii, only: whitespace
+    use stdlib_string_type, only: string_type, char, verify
+    use stdlib_optval, only: optval
     implicit none
-    private
+    private :: compute_lps
```

Comment:
Just leave the private statement here, it sets the default for all routines not explicitly declared
```suggestion
    private
```